src/Uno.UI/ts/Windows/Gaming/Input/Gamepad.ts

```diff
@@ -0,0 +1,82 @@
+﻿namespace Windows.Gaming.Input {
+	export class Gamepad {
+
+		private static dispatchGamepadAdded: (id: string) => number;
+		private static dispatchGamepadRemoved: (id: string) => number;
+
+		public static getConnectedGamepadIds(): string {
+			const gamepads = navigator.getGamepads();
+			const separator = ";";
+			var result = '';
+			for (var i = 0; i < gamepads.length; i++) {
+				if (gamepads[i]) {
+					result += gamepads[i].index + separator;
+				}
+			}
+			return result;
+		}
+
+		public static getReading(id: number): string {
+			var gamepad = navigator.getGamepads()[id];
```

Comment:
Codacy found an issue: [Identifier 'gamepad' is never reassigned; use 'const' instead of 'var'.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5499310)